package/base-files/files/lib/functions/network.sh

```diff
@@ -17,9 +17,9 @@ __network_ifstatus() {
 
 	__tmp="$(jsonfilter ${4:+-F "$4"} ${5:+-l "$5"} -s "${__NETWORK_CACHE:-{}}" -e "$1=@.interface${2:+[@.interface='$2']}$3")"
 
-	[ -z "$__tmp" ] && \
-		unset "$1" && \
-		return 1
+	[ -z "$__tmp" ] \
```

Comment:
One line I think.

Reply:
The condition has the first line, all subsequent things should have their own lines.